### PR TITLE
Ability to expand/collapse branches in layer tree

### LINKED ISSUE #892: Add ability to expand/collapse branches in layer tree
Related to https://github.com/camptocamp/c2cgeoportal/wiki/Spec-%231708-Layertree-desktop

Should only be available for depth > 1 for now.
Click on "chevron" icon to open or close group.

screenshot from 2016-03-16 10 02 46

### PR DESCRIPTION
**Pull request by olivierSemet**
closes #892

### CODE REVIEW

File: contribs/gmf/src/directives/partials/layertree.html [hunk 1]
```diff
@@ -1,4 +1,11 @@
 <span ng-if="::!layertreeCtrl.isRoot" id="node-{{::layertreeCtrl.uid}}" ng-class="[layertreeCtrl.node.children ? 'group' : 'leaf', 'depth-' + layertreeCtrl.depth, gmfLayertreeCtrl.getResolutionStyle(layertreeCtrl.node), gmfLayertreeCtrl.getNoSourceStyle(layertreeCtrl), gmfLayertreeCtrl.getNodeState(layertreeCtrl)]">
+    <a  ng-if="::layertreeCtrl.node.children"
+        data-toggle="collapse"
+        href="#layer-group-{{::layertreeCtrl.uid}}"
+        class="expand-node"
+        aria-expanded="{{::layertreeCtrl.depth == 1}}">
```
Comment (pgiraud) [inline contribs/gmf/src/directives/partials/layertree.html]: The initial state should not be addressed in this pull request.

**Comment by olivierSemet** [comment]
@pgiraud could check/confirm the PR?

File: contribs/gmf/less/layertree.less [hunk 2]
```diff
@@ -10,9 +12,19 @@ gmf-layertree ul .treenode {
     padding-left: 0;
     text-decoration: none;
     &::after {
-      display:none;
+      display: none;
+    }
+
+    &.expand-node.fa {
+      &:before {
```
Comment (pgiraud) [inline contribs/gmf/less/layertree.less]: `::before`
Though it works with a single ':', it's better to use the CSS3 notation with a double.

Comment (olivierSemet) [inline contribs/gmf/less/layertree.less]: Ok, fixed.

**Comment by pgiraud** [comment]
LGTM. Merging.

**Comment by fredj** [comment]
because the two icons don't have the same width, the text moves right when the group is expanded

**Comment by olivierSemet** [comment]
Yes agreed, I'm working on it

**Comment by fredj** [comment]
thanks

**Comment by olivierSemet** [comment]
Fixed

**Pull request merged**